processing/p5.js
# Update line normalization script to turn on normalization + fix line endings

Resolves https://github.com/processing/p5.js/issues/6207

### Changes:
- `text=auto` marks a file for line ending normalization. Without specifying `=auto`, it's unspecified, so it will only be normalized if the user's settings have that on globally, so the previous .gitattributes wasn't doing anything by default. (I suspect this was on globally when people were testing the original change to .gitattributes so we didn't notice.)
- Running `git add --renormalize .`, there were a few files with some different endings in the repo, so these got changed  for consistency.

To test this:
- I checked out the branch in https://github.com/processing/p5.js/pull/6276, where commit [db5bfea](https://github.com/processing/p5.js/pull/6276/commits/db5bfea46d033ac4f3f008813ba423f56f16769c) is currently failing with the linter check
- I ran `git rm --cached -r .; git reset --hard` to update the files in the working tree with new endings
- I then ran `npm run lint` to verify that the linting check passes again

#### PR Checklist

- [ ] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated
